calculator_scientific.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="calculator.css">
+    </head>
+    <body>
+        <div class="card rounded px-3 mx-auto">
+            <span><a href="calculator_standard.html">Standard</a> | Scientific</span>
+
+            <div class="row">
+                <div class="col-12">
+                    <form>
+                        <input id="inputSci" class="ml-1 rounded text-right">
+                    </form>
+                </div>
+                <div class="col-12">
+                    <button id="raddeg" class="bg-secondary m-1 rounded">Rad | Deg</button>
```

Comment:
This is fine but will have to be implemented as a toggle. More on that when we start JS